src/components/with-key-shortcuts/with-key-shortcuts.js

```diff
@@ -0,0 +1,117 @@
+import React, {
+  useCallback,
+  useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  checkScope,
+  HasCommand,
+} from '@folio/stripes-components';
+
+import { handleSaveKeyFormSubmit } from '../utilities';
+
+const WithKeyShortcuts = ({
```

Comment:
I'm not sure that this is a correct naming of this component, because naming started with word `with`  according throw naming [convention](https://airbnb.io/javascript/react/#naming) it should be a HOC, but here you have just component, so it could be better to name it `KeyShortcutsContaner`

Reply:
Okay, I can rename it. But I like more `KeyShortcutsWrapper`. Is it ok for you?

Reply:
Yeah, sure, that was just suggestion